generator/src/main.rs

```diff
@@ -0,0 +1,29 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+
+mod error;
+mod reports;
+
+use error::Error;
+
+#[derive(Parser)]
+struct Cli {
+    #[arg(short, long, env = "WEBSITE")]
+    website: PathBuf,
+    #[arg(short, long, env = "REPORTS")]
+    reports: String,
+}
+
+fn main() -> Result<(), Error> {
+    let cli = Cli::parse();
+    let reports_org_path = reports::clone(cli.reports)?;
+    let reports_org_list = reports::find(reports_org_path)?;
+    for report in reports_org_list {
+        match reports::convert(report, &cli.website) {
+            Ok(report_md) => {},
+            Err(e) => eprintln!("{:?}", e),
+        };
+    }
```

Comment:
I think we should just display the monthly and yearly reports on the website. Otherwise, this would create too much noise. We can probably do a `.filter()` and filter on the name list. And since we have to generate markdown headers which contain `yearly` or `monthly` tag, we can do it twice with a different filter and act accordingly.